src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,118 @@
+
 package core.basesyntax;
 
+import static java.lang.System.arraycopy;
+
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_VALUE = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int RESIZE_INDEX = 2;
+    private Node [] hashMap = new Node[DEFAULT_VALUE];
+    private int currentValue = DEFAULT_VALUE;
```

Comment:
Why do we need this variable? We can get this info through elements.length

Reply:
It's part of **resize** method